docs/content/development/extensions-contrib/google.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: doc_page
+---
+
+# Google Cloud Storage
+
+To use this extension, make sure to [include](../../operations/including-extensions.html) `druid-google-extensions` extension.
+
+## Deep Storage
+
+[Google Cloud Storage](https://cloud.google.com/storage/) is another option for deep storage. This requires some additional druid configuration.
+
+|Property|Description|Default|Required?|
+|--------|-----------|-------|---------|
+|bucket|Name of the Google Cloud bucket|N/A|yes|
+|path|The path where data is located.|N/A|yes|
```

Comment:
The correct property is `prefix` instead of `path`.

https://github.com/druid-io/druid/blob/7d36f540e809198206238c93f253ce54e4deed03/extensions-contrib/google-extensions/src/main/java/io/druid/storage/google/GoogleAccountConfig.java#L33